README.md

```diff
@@ -160,6 +160,13 @@ set -ag status-right "#{E:@catppuccin_status_session}"
 set -ag status-right "#{E:@catppuccin_status_uptime}"
 set -agF status-right "#{E:@catppuccin_status_battery}"
 
+# automatically set CWD as window name
+set -g @catppuccin_window_default_text "#W"
+set -g @catppuccin_window_current_text "#W"
+set -g @catppuccin_window_text "#W"
+set-option -g automatic-rename on
```

Comment:
Why automatic rename if `#W` is used? It updates automatically anyway

And please be consistent with `set` and `set-option`. They are aliases and `set` is used everywhere else